features/having_credit_memo_generated.feature

```diff
@@ -0,0 +1,34 @@
+@refunds
+Feature: Having credit memo generated
+    In order to have official confirmation of some order units refund
+    As an Administrator
+    I want to have credit memo generated for each order refund action
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And default tax zone is "US"
+        And the store has "US VAT" tax rate of 10% for "Clothes" within the "US" zone
+        And the store has a product "Mr. Meeseeks T-Shirt" priced at "$10"
+        And it belongs to "Clothes" tax category
+        And the store allows shipping with "Galaxy Post"
+        And the store allows paying with "Space money"
+        And this promotion gives "$1.00" off on every product with minimum price at "$5.00"
+        And there is a customer "rick.sanchez@wubba-lubba-dub-dub.com" that placed an order "#00000022"
+        And the customer bought 2 "Mr. Meeseeks T-Shirt" products
+        And the customer chose "Galaxy Post" shipping method to "United States" with "Space money" payment
+        And I am logged in as an administrator
+        And the order "#00000022" is already paid
+
+    @ui @application @todo
```

Comment:
Could we move `@ui @application` to the first line of the file?

Reply:
See https://github.com/Sylius/RefundPlugin/pull/13#discussion_r204004010 xD

Reply:
Still 😞